### DISCUSSION TITLE
[Feature Request] Character Counter for Textarea and Text Input

### DISCUSSION THREAD

**Discussion by soranoo** [heroui-inc/heroui#4748] [Feature requests]
### Is your feature request related to a problem? Please describe.

It would be nice to include a character counter to let the user know the limitations/current progress of the current field

### Describe the solution you'd like

Add an option to enable chara counter when the maxLength is set.

### Describe alternatives you've considered

N/A

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1200/[feature-request]-character-counter-for-textarea-and-text-input">ENG-1200 [Feature Request] Character Counter for Textarea and Text Input</a></p>

**Comment by ahmadJT** [comment]
@wingkwong Please can you assign this to me

**Comment by dantman** [comment]
It would be great to have this built in. As of now the Textarea doesn't even have a prop to insert content inside the textarea box that could be used to implement a counter.
